src/terrasoApi/account/accountService.ts

```diff
@@ -15,27 +15,24 @@
  * along with this program. If not, see https://www.gnu.org/licenses/.
  */
 import _ from 'lodash/fp';
-import * as terrasoApi from 'terrasoApi/terrasoBackend/api';
-import {
-  UserFragment,
-  UserPreferenceNode,
+import { graphql } from 'terrasoApi/gql';
```

Comment:
Question: I don't see this file in this code, is this generated by the `generate` command? If that is it, should we hook the generate command to other commands that need this file?

Reply:
yes, & the `generate` command needs to run for the client to build at all because of type checking. see https://github.com/techmatters/terraso-web-client/pull/934#discussion_r1195464222 for codegen discussion, https://github.com/techmatters/terraso-web-client/pull/934#discussion_r1195425986 for discussion of how to set up tooling